mmv1/third_party/terraform/website/docs/d/compute_instance_group_manager.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+subcategory: "Compute Engine"
+page_title: "Google: google_compute_instance_group_manager"
+description: |-
+  Get a Compute Instance Group within GCE.
+---
+
+# google\_compute\_instance\_group\_manager
+
+Get a Compute Instance Group Manager within GCE.
+For more information, see [the official documentation](https://cloud.google.com/compute/docs/instance-groups#managed_instance_groups)
+and [API](https://cloud.google.com/compute/docs/reference/latest/instanceGroupManagers)
+
+## Example Usage
+
+```hcl
+data "google_compute_instance_group_manager" "igm1" {
+  name = "k8s1-abcdef01-myns-mysvc-8080-4b6bac43"
```

Comment:
Why is this value used? It isn't wrong in any sense, just confusing a bit? why not "my-igm" or something simple

Reply:
updated